app/controllers/application_controller.rb

```diff
@@ -2,4 +2,16 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  NAVIGATION_BUTTONS= [
+    { text: 'OSRA', href: 'root_path', path_regex: /^\/$/, glyph: 'glyphicon-home' },
```

Comment:
I'd argue the Dashboard button does hold value, if you are at a business with confidential information on your screen and someone comes to talk to you, you can click it to show an OSRA logo instead of confidential information